src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1964,6 +1973,62 @@ private void ShowOpenDialogAndOpenResult(object parameter)
 
         private bool CanShowOpenDialogAndOpenResultCommand(object parameter) => CanRunGraph;
 
+        /// <summary>
+        /// Present the open dialog and open the template that is selected.
+        /// </summary>
+        /// <param name="parameter"></param>
+        private void ShowOpenTemplateDialog(object parameter)
```

Comment:
Any way we can combine this with `ShowOpenDialogAndOpenResult `?

Reply:
yeah I thought about this but we need to call `Open` function below with different tuple and use `TemplatesDirectory` as start directory, currently there is nothing we can use to differentiate two workflows other than use different command for each menu item. I can try think about this more